src/Menu/Matcher/Voter/AdminVoter.php

```diff
@@ -23,16 +24,42 @@
  */
 class AdminVoter implements VoterInterface
 {
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
     /**
      * @var Request
      */
     private $request = null;
 
     /**
+     * @param RequestStack $requestStack
+     */
+    public function __construct(RequestStack $requestStack = null)
+    {
+        $this->requestStack = $requestStack;
+    }
+
+    /**
+     * @deprecated since version 3.x. Pass a RequestStack to the constructor instead.
+     *
+     * @param Request $request
```

Comment:
Please remove this `@param`

Reply:
Why?

Reply:
See https://github.com/sonata-project/dev-kit/pull/354

Reply:
See also https://github.com/sonata-project/dev-kit/issues/345

Reply:
Sure, no problem